core/storage/user/gae_models.py

```diff
@@ -2507,6 +2516,10 @@ class PendingDeletionRequestModel(base_models.BaseModel):
 
     # The email of the user.
     email = datastore_services.StringProperty(required=True, indexed=True)
+    # Normalized username of the deleted user. May be None in the cases when
+    # the user was deleted after a short time and thus the username wasn't that
+    # known in the codebase.
+    normalized_username = datastore_services.StringProperty(indexed=True)
```

Comment:
Maybe normalized_long_term_username then, otherwise I would expect this to be always populated.

Reply:
Done.